ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -329,3 +329,4 @@ ansible-playbook -i $SHARED_DIR/inventory ~/fetch-kubeconfig.yml -vv || eval $PR
 ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $SHARED_DIR/inventory ~/fetch-information.yml -vv || eval $PROCEED_AFTER_FAILURES
 exit ${status}
 
+sleep 7200
```

Comment:
`sleep` should be before `exit`

Reply:
thanks Sagi!

Reply:
Updated, Also updated ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml , timeout was added in SNO job and not in MNO job, corrected.